styles/typography.less

```diff
@@ -18,3 +18,40 @@ a {
     color: @primary;
   }
 }
+
+/* Font-size for pages' h1 header; overwriting Bootstrap */
```

Comment:
We use `//` for single-line comments. (here and below)

Reply:
OK